test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -930,6 +930,82 @@ test.describe( 'Navigation block', () => {
 			await expect( linkControl.getSearchInput() ).not.toBeVisible();
 		} );
 	} );
+
+	test.describe( 'Navigation colors', () => {
+		test.beforeEach( async ( { requestUtils } ) => {
+			await Promise.all( [ requestUtils.deleteAllMenus() ] );
+		} );
+
+		test( 'As a user I expect my navigation links to inherit the colors from the theme', async ( {
+			admin,
+			editor,
+			requestUtils,
+		} ) => {
+			await admin.createNewPost();
+
+			await requestUtils.createNavigationMenu( {
+				title: 'Test Menu',
+				content:
+					'<!-- wp:navigation-submenu {"label":"First link","type":"custom","url":"http://www.wordpress.org/","kind":"custom","isTopLevelLink":true} --><!-- wp:navigation-link {"label":"Second Link","type":"custom","url":"http://www.wordpress.org/","kind":"custom","isTopLevelLink":true} /--><!-- /wp:navigation-submenu -->',
```

Comment:
Aside: didn't we ditch the `"isTopLevelLink":true` attribute? If so then we should remove from our tests 🙏

Reply:
Thank you for this ❤️ 

I believe @scruffian refactored away `isTopLevelLink` attribute when he was working on colors previously. He can confirm.

I was suggesting removing in order to avoid tests containing deprecated attributes.